src/link.js

```diff
@@ -5,7 +5,7 @@ module.exports = React.createClass({
     controller: React.PropTypes.object
   },
 
-  componentDidMount() {
+  componentWillMount() {
```

Comment:
please replace it with `componentWillMount: function () {`
I was hurry again :(
we need tests for repo :)